<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenu</class>
 <widget class="QDialog" name="GameMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>423</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Game menu</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>33</y>
     <width>190</width>
     <height>362</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Number of players:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="sb_numberOfPlayers">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Size of game board:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="sb_boardSize">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Number of cards:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="sb_numberOfCards">
      <property name="minimum">
       <number>12</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="singleStep">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btn_startGame">
      <property name="text">
       <string>Start new game</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Path to saved game:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="te_filePath"/>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="btn_choseFile">
      <property name="text">
       <string>Choose file...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="btn_loadGame">
      <property name="text">
       <string>Load game</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
